charts/images.yaml

```diff
@@ -415,6 +415,19 @@ images:
       confidentiality_requirement: 'high'
       integrity_requirement: 'high'
       availability_requirement: 'low'
+- name: init-large-dir
```

Comment:
What do you think about changing the name to make it more specific to its use, e.g. `loki-init-large-dir`?

Reply:
or maybe `tune2fs` because contains this binary?

Reply:
`tune2fs` fits best as it is the main binary of the container.